src/Dfe.PlanTech.Infrastructure.Data/Repositories/RecommendationsRepository.cs

```diff
@@ -79,7 +79,9 @@ await _db.RichTextContentWithSubtopicRecommendationIds
                 Header = intro.Header,
                 HeaderId = intro.HeaderId,
                 Maturity = intro.Maturity,
-                Content = introContent.Where(content => content.intro == intro.Id).Select(content => content.content!).ToList()
+                Content = [.. introContent.Where(content => content.intro == intro.Id)
+                                      .Select(content => content.content!)
+                                      .OrderBy(content => content.Order)]
```

Comment:
why the list copying [..
is it edited elsewhere?

Reply:
`introContent.Where(content => content.intro == intro.Id).Select(content => content.content!).OrderBy(content => content.Order)`

Returns an `IEnumerable` so needs to be cast to a `List` (or other concrete class). 

Originally I called `.ToList()` but the new preferred way (apparently) is this.